### PR TITLE
[EM-174] Update pull request size when base branch changes

### PR DESCRIPTION
**Pull request by SandroDamilano**
## What does this PR do?
It updates the value of the PR size metric every time the base branch of the PR changes.

Resolves [#174](https://rootstrap.atlassian.net/browse/EM-174)

### CODE REVIEW

**Review by GuilleLeopold — APPROVED** [review]

**Review by santib — APPROVED** [review]
minimalist 👏

File: spec/factories/payloads/pull_request.rb [hunk 1]
```diff
@@ -40,6 +40,34 @@
...
+    trait :edited do
+      transient do
+        changed { 'title' }
+        from { 'Old PR Title' }
+      end
+
+      action { 'edited' }
+      changes do
+        {
+          changed => {
```
Comment (santib) [inline spec/factories/payloads/pull_request.rb:52]:
```suggestion
          changed: {
```

Comment (SandroDamilano) [reply spec/factories/payloads/pull_request.rb:52]: It doesn't work the same way. I want the key to be whatever is in the `changed` variable (`'title'` is the default). If I use `changed:`, the key will be the symbol `:changed`.

Comment (santib) [reply spec/factories/payloads/pull_request.rb:52]: you are right, my bad 😄

**Pull request merged**